### PR TITLE
Feat/address-labels

### PR DESCRIPTION
**Pull request by seanblockchain**
## Description
1. Adds label input to Import Address form
2. Reformatting btc/bch imported addresses table
3. Adding labels to send dropdowns
4. Simplifying bch dropdown selector 
5. Small changes to relevant actions/sagas/tests 

## Testing Steps
1. Generate a bitcoin private key
2. Import Address in Settings->btc->imported-addresses (notice the new form input) 
3. Actions->edit-label (notice label change)
4. Send->bch->from (notice the labels in dropdown)

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
   reduce,
   set,
   sequence,
-  sort
+  sort,
+  lift,
+  assoc,
+  assocPath,
+  not,
+  isNil
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js]: Interesting diff

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js [hunk 2]
```diff
@@ -51,23 +51,28 @@ export const getData = (state, ownProps) => {
...
     excludeLockbox,
+    excludeWatchOnly,
     includeAll = true,
     includePitAddress
   } = ownProps
   const buildDisplay = wallet => {
+    const label =
+      wallet.label.length > 30
+        ? wallet.label.replace(/(.{17})..+/, '$1…')
+        : wallet.label
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js]: Nice, can you make this a utility function?

File: packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js (hunk 1 above)
Comment (schnogz) [reply packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js]: please keep these imported sorted by name

Comment (seanblockchain) [reply packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxBchAddresses/selectors.js]: I had taken those out previously when re-factoring the bch dropdown selector to more closely resemble the btc side. Realized I needed to filter out watch only addresses so had to put that functionality back.

Noted on alphabetical ordering of imports :  )

**Pull request merged**